src/main/java/frc/robot/utils/logging/commands/LoggableDeadlineCommandGroup.java

```diff
@@ -0,0 +1,48 @@
+package frc.robot.utils.logging.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.ParallelDeadlineGroup;
+import edu.wpi.first.wpilibj2.command.ProxyCommand;
+
+public class LoggableDeadlineCommandGroup extends ParallelDeadlineGroup implements Loggable {
+  private String basicName = getClass().getSimpleName();
+  private Command parent = new BlankCommand();
+
+  @SafeVarargs
+  public <T extends Command & Loggable> LoggableDeadlineCommandGroup(T deadline, T... others) {
```

Comment:
This is the first I've ever seen parameterized constructor like that. That's a neat trick